appveyor.yml

```diff
@@ -0,0 +1,89 @@
+version: '{branch}-{build}'
+build: off
+cache:
+  - '%LOCALAPPDATA%\pip\Cache'
+environment:
+  global:
+    WITH_COMPILER: 'cmd /E:ON /V:ON /C .\ci\appveyor-with-compiler.cmd'
+  matrix:
+    # - TOXENV: check
+    #   PYTHON: C:\Miniconda\python.exe
+    #   PYTHON_HOME: C:\Miniconda
+    #   PYTHON_VERSION: '2.7'
+    #   PYTHON_ARCH: '32'
+    - TOXENV: 'py27,report,codecov'
+      PYTHON: C:\Miniconda\python.exe
+      PYTHON_HOME: C:\Miniconda
+      PYTHON_VERSION: '2.7'
+      PYTHON_ARCH: '32'
+    - TOXENV: 'py27,report,codecov'
+      PYTHON: C:\Miniconda-x64\python.exe
+      WINDOWS_SDK_VERSION: v7.0
+      PYTHON_HOME: C:\Miniconda-x64
+      PYTHON_VERSION: '2.7'
+      PYTHON_ARCH: '64'
+    - TOXENV: 'py36,report,codecov'
+      PYTHON: C:\Miniconda36\python.exe
+      PYTHON_HOME: C:\Miniconda36
+      PYTHON_VERSION: '3.6'
+      PYTHON_ARCH: '32'
+    - TOXENV: 'py36,report,codecov'
+      PYTHON: C:\Miniconda36-x64\python.exe
+      PYTHON_HOME: C:\Miniconda36-x64
+      PYTHON_VERSION: '3.6'
+      PYTHON_ARCH: '64'
+    # There's no 3.7 package for PyAV yet - see https://anaconda.org/conda-forge/av/files
+    # - TOXENV: 'py37,report,codecov'
+    #   PYTHON: C:\Miniconda37\python.exe
+    #   PYTHON_HOME: C:\Miniconda37
+    #   PYTHON_VERSION: '3.7'
+    #   PYTHON_ARCH: '32'
+    # - TOXENV: 'py37,report,codecov'
+    #   PYTHON: C:\Miniconda37-x64\python.exe
+    #   PYTHON_HOME: C:\Miniconda37-x64
+    #   PYTHON_VERSION: '3.7'
+    #   PYTHON_ARCH: '64'
+init:
+  - "ECHO %PYTHON% %PYTHON_VERSION% %PYTHON_ARCH%"
+  - ps: echo $env:TOXENV
+  - "DIR %PYTHON_HOME%"
+install:
+
+  # Prepend to the PATH of this build
+  - "SET PATH=%PYTHON_HOME%;%PYTHON_HOME%\\Scripts;%PYTHON_HOME%\\Library\\bin;%PATH%"
+
+  # Install dependencies
+  - conda config --set always_yes true
+  # - conda update conda
+  - conda create -q -n test-environment python=%PYTHON_VERSION% pip virtualenv
+    numpy hdf5 h5py
```

Comment:
It's a library h5py uses. It should be pulled in by the h5py install; this was here in case we need to pin a version in the future, though with the new API that shouldn't be an issue any more!